### PR TITLE
Ms integration

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: storagestate/storagestate.go [hunk 1]
```diff
@@ -383,13 +383,10 @@ func (storageState *StorageState) StateAcceptance(updateState cloudprotocol.Stat
 	return nil
 }
 
-func (storageState *StorageState) GetInstanceCheckSum(instanceIdent aostypes.InstanceIdent) (string, error) {
-	state, ok := storageState.statesMap[instanceIdent]
-	if !ok {
-		return "", ErrNotFound
-	}
+func (storageState *StorageState) GetInstanceCheckSum(instanceIdent aostypes.InstanceIdent) string {
+	state := storageState.statesMap[instanceIdent]
```
Comment (al1img) [inline storagestate/storagestate.go]: But if there is no state for this instance, should we return error?

File: iamclient/iamclient.go [hunk 2]
```diff
@@ -192,15 +192,14 @@ func (client *Client) InstallCertificates(
 		response, err := client.certificateService.ApplyCert(ctx, request)
 		if err == nil {
 			certConfirmation.Status = "installed"
+			certConfirmation.Serial = response.Serial
```
Comment (al1img) [inline iamclient/iamclient.go:195]: What is the problem with filling serial all the time?

Comment (Eabramov84) [reply iamclient/iamclient.go:195]: response is nil

**Review by al1img — APPROVED** [review]

File: storagestate/storagestate.go (hunk 1 above)
Comment (Eabramov84) [reply storagestate/storagestate.go]: it is not ana error because instance could doesn't;t have ana state

**Pull request merged**